code/datums/traits/negative.dm

```diff
@@ -38,9 +38,17 @@
 	var/obj/item/heirloom_type
 	switch(quirk_holder.mind.assigned_role)
 		if("Clown")
-			heirloom_type = /obj/item/bikehorn/golden
+			heirloom_type = /obj/item/paint/anycolor
 		if("Mime")
-			heirloom_type = /obj/item/reagent_containers/food/snacks/baguette
+			heirloom_type = /obj/item/toy/dummy
+		if("Cook")
+			heirloom_type = /obj/item/kitchen/knife
+		if("Medical Doctor")
+			heirloom_type = /obj/item/reagent_containers/hypospray
+		if("Station Engineer")
+			heirloom_type = /obj/item/wirecutters/brass
```

Comment:
No why are you giving them clockcult wirecutters

Reply:
There same as normal

Reply:
Brass cutters are faster and would cause a scare for cult activity unless someone was smart enough to pick up that they're an heirloom item. Seems like a bad idea for balance but it might be funny because it would put the scare into AIs. @kevinz000